clang/unittests/Format/FormatTest.cpp

```diff
@@ -11077,6 +11077,157 @@ TEST_F(FormatTest, WrapsTemplateDeclarationsWithComments) {
       Style);
 }
 
+TEST_F(FormatTest, BreakBeforeTemplateClose) {
```

Comment:
I want to see some lambdas and template usages (not declarations).

Reply:
Good suggestion! I have added tests in this commit: https://github.com/llvm/llvm-project/pull/118046/commits/3927d411eeece27521e1a4034d01012c84beaf4b